src/pretix/plugins/sendmail/views.py

```diff
@@ -36,13 +41,41 @@ def form_valid(self, form):
         self.request.event.log_action('pretix.plugins.sendmail.sent', user=self.request.user, data=dict(
             form.cleaned_data))
 
+        tz = pytz.timezone(self.request.event.settings.timezone)
+
         failures = []
+        self.output = {}
         for o in orders:
-            try:
-                mail(o.email, form.cleaned_data['subject'], form.cleaned_data['message'],
-                     None, self.request.event, locale=o.locale, order=o)
-            except SendMailException:
-                failures.append(o.email)
+            if self.request.POST.get("action") == "preview":
+                for l in self.request.event.settings.locales:
+                    with language(l):
+                        self.output[l] = []
+                        self.output[l].append(
+                            form.cleaned_data['subject'].localize(l))
```

Comment:
uh, I see. You are using the `ugettext_lazy` that you showed me the other day... Well, I wouldn't have thought of that! Looks much better :)